editor/components/writing-flow/index.js

```diff
@@ -140,6 +140,12 @@ class WritingFlow extends Component {
 		this.props.onMultiSelect( currentStartUid, blocks[ nextIndex ] );
 	}
 
+	moveSelection( blocks, currentUid, delta ) {
+		const currentIndex = blocks.indexOf( currentUid );
+		const nextIndex = Math.max( 0, Math.min( blocks.length - 1, currentIndex + delta ) );
```

Comment:
Minor: could use `lodash#clamp`.

Reply:
TIL! Changed in 548d23115c5432ce0ec37371943108187b4d038c.